positive_or_negative.c

```diff
@@ -0,0 +1,20 @@
+#include <stdio.h>
```

Comment:
Once again, did you run this code? It's full of syntax errors. Please see us ASAP.